pkgs/build-support/fetchmavenartifact/default.nix

```diff
@@ -0,0 +1,57 @@
+# Adaptation of the MIT-licensed work on `sbt2nix` done by Charles O'Farrell
+
+{ fetchurl, stdenv }:
+let
+  defaultRepos = [
```

Comment:
It'd be good if this were an attribute somewhere so we could append to it, not just override it.